### ISSUE TITLE
Lesson/review breakdown incorrect titles, colors not matching and missing values

### ISSUE THREAD

**Issue by MTLVMPR** [jerryhcooke/smouldering_durtles#4]
It shows 27 Kana lessons and 121 Kana reviews when those shouldn't even exist yet. (They are just the vocab items)

Only the top row has the numbers colored as well. 

Earlier level lessons are missing too.

![image](https://github.com/jerryhcooke/smouldering_durtles/assets/93704391/7e7520bc-eda4-47ed-8902-d147829bd73a)

**Comment by jerryhcooke** [comment]
Okay, I've resolved this, I believe, and will post a test build soon. 

The issue was due to this section of code (replicated similarly in each of the four rows:

```
            }
            if (subject.getLevel() < userLevel && subject.getType().isVocabulary()) {
                reviewPastVocCount++;
            }
            if (subject.getLevel() < userLevel && subject.getType().isVocabulary()) {
                reviewPastKanaVocCount++;
            }
```

I made these changes before realising Tofugu's intent to lump kana-only vocab in with regular vocab. 

I've removed the headers for kana in that table and have modified the four code blocks as shown below, which should cause kana_vocab items to be added to the vocabulary count on that display. 

```
            }
            if (subject.getLevel() < userLevel && subject.getType().isVocabulary()) {
                reviewPastVocCount++;
            }
            if (subject.getLevel() < userLevel && subject.getType().isKanaVocabulary()) {
                reviewPastVocCount++;
            }

```

**Comment by jerryhcooke** [comment]
While this appears to correct the vocab count, there is an unresolved issue with kanji adding to the vocab count, despite also displaying their own numbers. 

Have done some database rewrites and testing different numbers. 

This 2x Ra, 2x Ka, 2x Vo, 2x Kv
Shows as 2 Ra, 2 Ka, 6 Vo

This 2x Ra, 0x Ka, 2x Vo, 2x Kv
Shows as 2 Ra, 0 Ka, 4 Vo

Adjusting numbers of Ra, Vo, Kv has no incorrect totals, but adding Kanjis always adds to the Vo total.

**Comment by jerryhcooke** [comment]
Also fixed issue with the timeline when set in subject type mode not showing correct numbers, will push fix with this, test, and then release a package.

**Comment by jerryhcooke** [comment]
Issues confirmed resolved in v1.0.1
